cmd/genproto/genproto.go

```diff
@@ -24,6 +29,22 @@ import (
 )
 
 func main() {
+	cmd := commands.NewCommand(
+		commands.Metadata{
+			LongHelp: "Generates proto bindings for Amino packages",
+		},
+		nil,
```

Comment:
Use an empty structure instead of nil; just to make it more clear that it’s where we can pass the options.